search/templates/cosmos/searchresults.html

```diff
@@ -3,8 +3,12 @@
   <title>"{{ query }}" | cosmos-search</title>
 {% endblock %}
 {% block body %}
-<h6>Showing <span style="color: #5D337F;">{{ amount }}</span> results for: <span style="color: #5D337F;">{{ query }}</span>
-<span>
+    {% if amount > 1 %}
```

Comment:
If there are 0 results, then this would not work. The grammatically correct output would be "0 results". As such, the code should be:

```HTML
{% if amount == 1 %}
        <h6>Showing <span style="color: #5D337F;">1</span> result for: <span style="color: #5D337F;">{{ query }}</span>
    {% else %}
        <h6>Showing <span style="color: #5D337F;">{{ amount }}</span> results for: <span style="color: #5D337F;">{{ query }}</span>
    {% endif %}
```

Reply:
Never mind, this doesn't apply, since we have a 404 page. 😐 

<sub>Sometimes I get ahead of myself. \*sigh*</sub>